src/stdlib/base.bt

```diff
@@ -4,7 +4,7 @@
 // :param (string) $msg: The message to print if the condition is not met
 macro assert($cond, $msg) {
   if (!$cond) {
-    printf("Assertion failed. Msg: %s", $msg);
+    errorf("Assertion failed. Msg: %s", $msg);
```

Comment:
Good thing it says `Msg: ` here, I never would have known that it's the message ;)

Why not just `assert failed: %s`?

Reply:
I like beating people over the head but yes, I'll change it so as not to be a complete hypocrite.